### ISSUE TITLE
Missing documentation for package operators and public classes

### ISSUE THREAD

**Issue by glopesdev** [neurogears/onix-refactor#173]
While reorganizing the project for packaging, I noticed the following operators and types remain undocumented:

- [`ContextTask`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/ContextTask.cs)
- [`CreateContext`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/CreateContext.cs)
- [`ConfigureHarpSyncInput`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/ConfigureHarpSyncInput.cs)
- [`ConfigureLoadTester`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/ConfigureLoadTester.cs)
- [`DeviceFactory`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/DeviceFactory.cs)
- [`HarpSyncInputData`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/HarpSyncInputData.cs)
- [`HarpSyncInputDataFrame`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/HarpSyncInputDataFrame.cs)
- [`HubDeviceFactory`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/HubDeviceFactory.cs)
- [`PassthroughState`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/PassthroughState.cs)
- [`StartAcquisition`](https://github.com/neurogears/onix-refactor/blob/6dc9e26d38a48b9a0f5c13ffb0a8966236ba2144/OpenEphys.Onix/OpenEphys.Onix/StartAcquisition.cs)

We could potentially internalize some of these as implementation-specific classes, which would simplify both the API and docs effort at this point, but the operators will need to be included.

**Comment by jonnew** [comment]
Yep on my to-do tomorrow. Thanks for pulling out the missing ones.

**Comment by glopesdev** [comment]
@jonnew great, let me know if you want to merge #172 to main beforehand so you have a few less classes to worry about.

**Comment by jonnew** [comment]
That would be helpful. Im going to fork from 172 for my working copy

**Comment by glopesdev** [comment]
Great, no need to fork, I have merged it now.
